src/modules/fw_pos_control_l1/fw_pos_control_l1_params.c

```diff
@@ -1093,3 +1110,15 @@ PARAM_DEFINE_INT32(FW_LND_ABORT, 3);
  * @group FW TECS
  */
 PARAM_DEFINE_FLOAT(FW_WIND_ARSP_SC, 0.f);
+
+/**
+ * FW Launch detection
+ *
+ * Enables automatic launch detection based on measured acceleration. Use for hand- or catapult-launched vehicles.
+ * Only available for fixed-wing vehicles.
+ * Not compatible with runway takeoff.
+ *
+ * @boolean
+ * @group FW Launch detection
+ */
+PARAM_DEFINE_INT32(LAUN_DETCN_ON, 0);
```

Comment:
FW prefix?

Reply:
Alright, renamed to FW_LAUN_DCTN_ON.
And to be consistent also changed the other 3 LAUN_* params. Incl. removing the CAT from the name, as it's a generic launch detector, not limited to work on a catapult.